pages.zh/linux/ul.md

```diff
@@ -0,0 +1,13 @@
+# ul
+
+> 执行文本的下划线。
+> 给定字符串中的每个字符必须单独加下划线。
+> 更多说明请访问： <https://manned.org/ul>.
+
+- 在适用的情况下用下划线显示文件的内容：
+
+`ul {{file.txt}}`
```

Comment:
```suggestion
ul {{path/to/file.txt}}
```

Reply:
Please translate the `path/to/file` for get this easier for non-English speakers.

Reply:
@navarroaxel fixed.please check